.travis.yml

```diff
@@ -9,10 +10,11 @@ addons:
     - ubuntu-toolchain-r-test
     packages:
     - g++-5
+    - clang
 
 install:
 - export CXX="g++-5"
-- script/configure
+- scan-build script/configure
```

Comment:
`scan-build` sets `$CC` and `$CXX` to be the `ccc-analyzer` script which `gyp` will then pick up. When `make` runs, it invokes `ccc-analyzer` for each object, which will then invoke `g++-5` and `clang++`.

It's kind ugly but [is the recommended way](https://clang-analyzer.llvm.org/scan-build.html#recommended_autoconf)  😐